setup.py

```diff
@@ -44,14 +44,17 @@
         'requests',
         'tensorflow-federated',
     ],
-    # TensorFlow doesn't yet support Python 3.9.
-    python_requires='>=3.6, <3.9',
+    # TensorFlow support Python 3.9.
```

Comment:
Could you simply remove this comment as it is no longer necessary?

Reply:
Done!